### ISSUE TITLE
is it possible to set Hour & Minute by input text ?

### ISSUE THREAD

**Issue by pramahaditamaputra** [rehookify/datepicker#72]
<img width="311" alt="Screenshot 2024-09-17 at 10 54 56" src="https://github.com/user-attachments/assets/0c335848-2040-4a8d-b410-2e401472ac21">

is it possible to set Hour & Minute by input text ? the current API Provided it just return array of time

**Comment by ceoshikhar** [comment]
Yes man. Even I wanted to implement this way so that user can input a time with an interval of 1min but rendering the list with 1min interval is VERY bad UX.


@Feshchenko Any help on this will be greatly appreciated man! <3 Thank you
